idl/chromadb/proto/coordinator.proto

```diff
@@ -175,14 +175,18 @@ message GetCollectionResponse {
   Collection collection = 1;
 }
 
+message CollectionIdsFilter {
+  repeated string ids = 1;
+}
+
 message GetCollectionsRequest {
   optional string id = 1;
   optional string name = 2;
   string tenant = 4;
   string database = 5;
   optional int32 limit = 6;
   optional int32 offset = 7;
-  repeated string ids = 8;
+  optional CollectionIdsFilter ids_filter = 8;
```

Comment:
this wrapper type is unfortunately needed because otherwise we can't distinguish the case when no IDs filter is provided and when an empty IDs filter (`[]`) is provided

Reply:
But is it backwards and forwards compatible? For e.g. if old sysdb receives the new type, it will error out trying to deserialize this field (vice-versa)

Reply:
discussed offline--yes, it's ok as long as the change that introduced `GetCollectionsRequest.ids` doesn't get promoted to prod separately from this change